perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -210,23 +222,31 @@ def _perform_testing(
                         (all samples) for each testing folder.
     persistent_disk_results: A dictionary containing the list of results
                              (all samples) for each testing folder.
+    start_and_end_times_ms_persistent_disk: A dictionary containing the list of
+                                            start and end times (all samples) for
+                                            each testing folder.
+    start_and_end_times_ms_gcs_bucket: A dictionary containing the list of
+                                       start and end times (all samples) for
+                                       each testing folder.
   """
 
   gcs_bucket_results = {}
   persistent_disk_results = {}
+  start_and_end_times_persistent_disk = {}
```

Comment:
We don't need to create two more dicts and just return list of (start_time, end_time) tuples from which we can calculated latencies in the downstream code.